diesel_tests/Cargo.toml

```diff
@@ -6,11 +6,11 @@ license = "MIT OR Apache-2.0"
 build = "build.rs"
 
 [build-dependencies]
-syntex = { version = "0.42.0", optional = true }
+syntex = { version = ">= 0.43.0", optional = true }
 diesel_codegen_syntex = { path = "../diesel_codegen_syntex", optional = true }
-dotenv_codegen = { version = "0.9.0",  optional = true }
+dotenv_codegen = { git = "https://github.com/slapresta/rust-dotenv", version = "0.9.3",  optional = true }
```

Comment:
It has [not been released](https://crates.io/search?q=dotenv). As such, we'll need to point to Git for the time being.